hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
Oh, I mixed up the two files here.
Never the less there is a possibility to exit ```http_utils.fetch``` without closing the connection:
an ```asyncio.CancelledError``` between creating and reading from the response.
To outline this, I created a small tester:
https://gist.github.com/das7pad/e8f63175377576a0cb3466d20a417fa5

I added a lag between creating and processing the response to raise the ```asyncio.CancelledError``` in the small timespawn.